validation/src/main/scala/hmda/validation/rules/lar/quality/Q046.scala

```diff
@@ -0,0 +1,17 @@
+package hmda.validation.rules.lar.quality
```

Comment:
This file has a strange git history. `git log --follow validation/src/main/scala/hmda/validation/rules/lar/quality/Q046.scala` will show you what I mean.  Started a discussion in chat, since it's not specific to this issue, but here's an example that is not yet merged.